anti-analysis/anti-debugging/debugger-detection/check-for-peb-ntglobalflag-flag.yml

```diff
@@ -4,7 +4,9 @@ rule:
     namespace: anti-analysis/anti-debugging/debugger-detection
     authors:
       - moritz.raabe@mandiant.com
-    scope: function
+    scopes:
+      static: function
+      dynamic: process
```

Comment:
this rule can't match in dynamic analysis because it relies on an offset feature being present.

Reply:
it also relies on the basic block subscope matching, which also doesn't work in dynamic analysis. so, i think the rule should be marked as unsupported in dynamic scope.